spec/prest_spec.rb

```diff
@@ -98,6 +108,23 @@
                                                                                                   headers: {})
           subject
         end
+
+        include_examples 'returns a correct response'
+      end
+
+      describe 'accessing the response params' do
+        let!(:parsed_response) { { key: 'value' } }
+        let!(:headers) { { 'Content-Type' => 'application/json' } }
+
+        include_examples 'returns a correct response'
+
+        include_examples 'allows access to response data'
```

Comment:
am not sure if allows is the correct word for this shared example. WDYT about renaming it to `"forwards methods to httparty's response"`